container/render.py

```
@@ -132,6 +133,24 @@ def validate_args(args):
     )
 
 
+def get_commit_sha():
+    try:
+        result = subprocess.run(
+            ["git", "rev-parse", "HEAD"],
```

Ruff (0.15.4)

[error] Starting a process with a partial executable path

(S607)